xl_auth/user/views.py

```diff
@@ -90,7 +90,11 @@ def view(user_id):
         flash(_('User ID "%(user_id)s" does not exist', user_id=user_id), 'danger')
         return redirect(url_for('user.profile'))
     else:
-        return render_template('users/view.html', user=user)
+        # Cataloging admins and admins need to see more information, so they get their own view.
+        if current_user.is_admin or len(current_user.get_cataloging_admin_permissions()) > 0:
```

Comment:
@mxtthias We already have a `is_cataloging_admin` property, it seems this new method (and the use here) is not needed.

Reply:
Good catch! I changed to `current_user.is_cataloging_admin` in `views.py`, but we use `get_cataloging_admin_permissions` in the templates (see https://github.com/libris/xl_auth/pull/118/files#diff-70b4b59a3969baa75d0676efb0f2900dR16). I felt it would make the templates simpler. Are you OK with keeping it?